projects/ti/cc3220_launchpad/iar/aws_demos/aws_demos.ewp

```diff
@@ -2428,11 +2420,26 @@
 					<name>$PROJ_DIR$\..\..\..\..\..\libraries\3rdparty\pkcs11\pkcs11t.h</name>
 				</file>
 			</group>
+			<group>
+				<name>mbedtls_utils</name>
+				<file>
+					<name>$PROJ_DIR$\..\..\..\..\..\libraries\3rdparty\mbedtls_utils\mbedtls_utils.c</name>
+				</file>
+				<file>
+					<name>$PROJ_DIR$\..\..\..\..\..\libraries\3rdparty\mbedtls_utils\mbedtls_error.h</name>
```

Comment:
I think this should be separated into an include sub-group and the .c files into a src sub-group..